### ISSUE TITLE
Proxy Voting

### ISSUE THREAD

**Issue by lkngtn** [aragoncoop/ideas#5]
It would be useful for ANT holders to be able to deposit ANT into a proxy voting contract controlled by the cooperative. 

The contract should allow anyone to deposit ANT and withdraw at any time. It should allow a designated address to use the ANT in the contract for the purpose of voting, but should not allow the designated address to transfer ANT.

**Comment by yeqbfgxjiq** [comment]
Would there be multiple proxy voting contracts? For example if I know I generally agree with someone, could I delegate specifically to them and "follow" their votes and give them more leverage?

Or are you saying that coop members would vote internally on how the Coop (not each member with their personal tokens, but the organization) will vote in upcoming AGPs, and people could "follow" the org and throw their weight behind it?

**Comment by lkngtn** [comment]
If the contract is built it could be deployed multiple times with different designated addresses, but I think it needs to be one contract per designated address.

**Comment by 0xGabi** [comment]
Great idea. It would be amazing to use `dao act` on this addresses so coop members can vote during AGPs with those tokens

**Comment by lkngtn** [comment]
Yeah we could potentially also do an AGP that asks the network to put a certain amount in coops proxy contract. So it wouldnt be a grant of ANT, but it would give the coop more direct influence in the AGP process.

**Comment by yeqbfgxjiq** [comment]
So rather than having all treasury funds sitting in vaults quarantined from voting on AGPs, some of it could be released to secure "voting/proxy vaults/contracts" that Aragon ecosystem communities could then use to leverage their voice in the AGP process, but not directly access the funds?
^^ Sorry I read this pre-coffee. @lkngtn You literally just said this in your comment above. My bad lol

**Comment by sepu85** [comment]
just passing by to say I love this idea!
